gef-ui/component-tests/partials/facility-end-date.component-test.js

```diff
@@ -0,0 +1,138 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = 'partials/facility-end-date.njk';
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+
+  const dealId = '61e54dd5b578247e14575882';
+  const facilityId = '666862d9140a08222cbd69e7';
+
+  const aFacilityEndDateError = (message, refs) => {
+    const fieldErrors = {
+      facilityEndDate: {
+        text: message,
+      },
+    };
+
+    refs.forEach((ref) => {
+      fieldErrors[`facilityEndDate-${ref}`] = {
+        text: message,
+      };
+    });
+
+    return {
+      errorSummary: message,
+      fieldErrors,
+    };
+  };
+
+  const params = {
+    dealId,
+    facilityId,
+  };
+
+  beforeEach(() => {
+    wrapper = render(params);
+  });
+
+  it(`only renders the 'Back to previous page' link if the status does not equal 'change'`, () => {
```

Comment:
I think this might read better as `does  not render the 'Back to previous page' link when the status is 'change'` as i had to double take for some odd reason... 

Also this pattern is slightly surprising where we test two things each test -- is this an existing pattern? I kind of don't like it but see what it's trying to do.

Reply:
I think I took this pattern from Oscar's component tests, have separated the two cases